lighty-examples/lighty-controller-springboot-netconf/src/main/java/io/lighty/core/controller/springboot/config/LightyConfiguration.java

```diff
@@ -111,13 +115,26 @@ void shutdown() {
     void closeLightyModule(final LightyModule lightyModule) {
```

Comment:
`private static`

Reply:
Also change line 105 to:
`Runtime.getRuntime().addShutdownHook(new Thread(this::shutdown));`